### PR TITLE
Issue 6168. Remove $form['#attributes']['class'] assignment in `installer_browser_install_button_form()`

### LINKED ISSUE #6168: installer_browser_install_button_form() assigns a string to `$form['#attributes']['class']`
## Description of the bug

The form callback `installer_browser_install_button_form()` assigns a string to `$form['#attributes']['class']` instead of an array.

While this doesn't cause any immediate issue with this simple form (which basically provides the "Install" button once you select a project to install), it's inconsistent with all other core assignments of  `$form['#attributes']['class']` or `$element['#attributes']['class']`, which is always an array of classes. 

## Steps To Reproduce

There is no error or warning generated in core, but this was found because of a [contrib project](https://github.com/backdrop-contrib/scheduler/issues/22) that tried to do something like:

`if (in_array('node-form', $form['#attributes']['class'])) {}`

inside a `hook_form_alter()` implementation. This kind of check is very common in core for class attributes (it's kind of a miracle that this check is not done to this particular form - otherwise we would have seen the error).

If you wish to verify that `$form['#attributes']['core']` is a string after this form, you can implement a `hook_form_alter()` and output `dpm($form['#attributes']['class']);`  for every form. Then visit `admin/modules/install` and select a project to install, so that the "Install" button is shown.

## Actual behavior

The assignment is a string.

## Expected behavior

The assignment should be an array (of classes).

## Additional information

- Backdrop CMS version: 1.25.1

------

PR https://github.com/backdrop/backdrop/pull/4471

### PR DESCRIPTION
**Pull request by argiepiano**
Fixes https://github.com/backdrop/backdrop-issues/issues/6168

### CODE REVIEW

File: core/modules/installer/installer.pages.inc [hunk 1]
```diff
@@ -846,7 +846,7 @@ function installer_browser_get_install_list() {
  */
 function installer_browser_install_button_form($form, &$form_state) {
   $project_type = installer_browser_type_from_path();
-  $form['#attributes']['class'] = 'installer-browser-install-button-form';
+  $form['#attributes']['class'] = array('installer-browser-install-button-form');
```
Comment (indigoxela) [inline core/modules/installer/installer.pages.inc]: Funny, I just realized, that the line isn't necessary at all.

That class gets set anyway in function backdrop_retrieve_form, via `$form['#attributes']['class'][] = backdrop_html_class($form_id);`

Comment (argiepiano) [reply core/modules/installer/installer.pages.inc]: Interesting! Let's remove that, then!

Comment (djzwerg) [reply core/modules/installer/installer.pages.inc]: I can confirm that removing this line works too 😄

**Review by djzwerg — APPROVED** [review]
Removing this line makes https://github.com/backdrop-contrib/scheduler/issues/22 to have been gone.

**Comment by laryn** [comment]
Thanks to 
 - @onyxnz for the initial discovery
 - @argiepiano for tracking down and fixing
 - @indigoxela, @herbdool, @djzwerg for testing and review!

Merged into `1.x` and `1.25.x`.

**Pull request merged**